docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -64,54 +72,73 @@ $ sudo snap set edgexfoundry support-notifications=off
 All services which are installed on the system as systemd units, which if enabled will automatically start running when the system boots or reboots.
 
 ## Configuring individual services
+The EdgeX snaps support configuration overrides via snap configure hooks which generate service-specific .env files which are used to
+provide a custom environment to the service, overriding the default configuration provided by the service's ```configuration.toml```
+file. If a configuration override is made after a service has already started, then the service must be **restarted** via command-line
+(e.g. ```snap restart edgexfoundry.<service>```), snapd's REST API, or the SMA (sys-mgmt-agent). If the overrides are provided via the
```

Comment:
Do we want to even mention `or the SMA (sys-mgmt-agent).` ? Since it is deprecated and I thought was enabled by default.

Reply:
I removed the reference